prow/config/tide.go

```diff
@@ -98,11 +102,19 @@ func (tq *TideQuery) Query() string {
 	return strings.Join(toks, " ")
 }
 
-// AllPRs returns all open PRs in the repos covered by the query.
-func (tq *TideQuery) AllPRs() string {
+// AllPRsSince returns all open PRs in the repos covered by the query that
+// have changed since time t.
+func (tq *TideQuery) AllPRsSince(t time.Time) string {
 	toks := []string{"is:pr", "state:open"}
 	for _, r := range tq.Repos {
 		toks = append(toks, fmt.Sprintf("repo:\"%s\"", r))
 	}
+	if t.Year() < 1970 {
```

Comment:
Do you seriously expect `t.Year() < 1970 && !t.IsZero()`? This should just be `if !t.IsZero() { toks = append(...) } `

Reply:
That is what I had originally, but this is more robust. If the caller of `AllPRsSince` uses a time value earlier than 1970 this will log an error instead of silently failing to find PRs (the graphql API does not return an error if you use an invalid time string).

Reply:
I simplified this to just check that the time has year >= 1970.